common/src/types/v0/transport/volume.rs

```diff
@@ -425,14 +425,22 @@ pub struct RepublishVolume {
     pub target_node: Option<NodeId>,
     /// Share protocol.
     pub share: VolumeShareProtocol,
+    /// Republish without reusing older nexus.
+    pub force: bool,
 }
 impl RepublishVolume {
     /// Create new `RepublishVolume` based on the provided arguments.
-    pub fn new(uuid: VolumeId, target_node: Option<NodeId>, share: VolumeShareProtocol) -> Self {
+    pub fn new(
+        uuid: VolumeId,
+        target_node: Option<NodeId>,
+        share: VolumeShareProtocol,
+        force: bool,
```

Comment:
can we use something else rather than "force", maybe `reuse_existing` or something like that?

Reply:
Yeah.